.claude/agents/test-reviewer.md

```
@@ -52,6 +52,12 @@ tools: Read, Grep, Glob
 - [ ] Fixture만으로 표현이 어려운 케이스는 테스트 안에서 빌더를 만들지 말고 `AlarmOccurrenceFixture.toEntity(...)` 같은 오버로드를 Fixture에 추가했는가?
 - [ ] 테스트 본문에서는 `AlarmFixture.ALARM_01.toEntity(...)`, `AlarmOccurrenceFixture.ALARM_OCCURRENCE_01.toEntity(...)`처럼 Fixture 호출 형태를 우선 사용했는가?
 
+### 에러 응답 검증
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)